plugins/woocommerce/src/Internal/ProductAttributesLookup/LookupDataStore.php

```diff
@@ -703,11 +745,287 @@ private function add_product_attributes_lookup_table_settings( array $settings,
 					'type'          => 'checkbox',
 					'checkboxgroup' => 'start',
 				);
+
+				$settings[] = array(
+					'title'         => __( 'Optimized updates', 'woocommerce' ),
+					'desc'          => __( 'Use direct database access to retrieve the data needed for updates, instead of the standard WordPress and WooCommerce data store functions.', 'woocommerce' ),
```

Comment:
```suggestion
					'desc'          => __( 'Uses much more performant queries to update the lookup table, but may not be compatible with some extensions.', 'woocommerce' ),
```

Suggested change to give users more info about why they might want to check the box, or not.